Analytics/App_Plugins/Analytics/backOffice/AnalyticsTree/directives/localizeOptions.Directive.js

```diff
@@ -0,0 +1,12 @@
+﻿angular.module("umbraco.directives").directive('localizeOption', function (localizationService) {
+    return {
+        restrict: 'A',
+        link: function ($scope, element, attrs, controller) {
+            var key = attrs.localizeOption;
+
+            localizationService.localize(key).then(function (value) {
+                element.text(value);
+            });
+        }
+    }
+});
```

Comment:
@warrenbuckley haven't looked in source of Umbraco to see if it has a directive for this - but it does use Umbraco's localizationService. Just saw that forum post how to solve the translations for option values in a dropdownlist.